libflux/flux-core/src/bin/fluxdoc.rs

```diff
@@ -237,7 +237,7 @@ impl<'a> example::Executor for CLIExecutor<'a> {
 // high standard going forward.
 //
 // See https://github.com/influxdata/flux/issues/4141 for tacking removing of this list.
-const EXCEPTIONS: [&str; 97] = [
+const EXCEPTIONS: [&str; 94] = [
```

Comment:
```suggestion
const EXCEPTIONS: &[&str] = &[
```

Reply:
Thank you 🙏

Reply:
@sanderson No need to update array length after this.